### DISCUSSION TITLE
Problemas com o snippet do html5 no VScode

### DISCUSSION THREAD

**Discussion by sarahdialmeida** [frontendbr/forum#2009] [Perguntas e Respostas]
Oi pessoal, 
o Snippet inicial do html:5 resolveu não funcionar no VsCode em um projeto especifico. Escrevo no editor html e normalmente ele já auto completa pro html:5 
```
<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>Document</title>
</head>
<body>
	
</body>
</html>
```

Por indicação de um amigo desabilitei todas as extensões, também já verifiquei se o VSCode está atualizado e experimentei em outros projetos e sim funcionam. Pensei que poderia ter feito algo errado ao clonar e repeti usando outro processo e ainda assim segue dando erro. O que vocês acham que pode ser?

**Comment by vitebo** [comment]
pode ser alguma config do vscode desse projeto em especifico.
verifica se existe uma pasta `.vscode` na raiz do projeto ~(não tenho certeza se o nome é esse)~
se tiver provavelmente tem alguma config desabilitando isso la dentro

**Reply by sarahdialmeida** [reply]
A janela do snippet estava reduzida e ocultando as demais opçoes de html. Obrigada pela ajuda @vitebo !

**Reply by vitebo** [reply]
Boa! que bom que conseguiu dar um jeito aí 😀
